pkg/backup/actions/csi/pvc_action.go

```diff
@@ -563,3 +571,61 @@ func NewPvcBackupItemAction(f client.Factory) plugincommon.HandlerInitializer {
 		}, nil
 	}
 }
+
+func setPVCRequestSizeToVSRestoreSize(
+	pvc *corev1api.PersistentVolumeClaim,
+	crClient crclient.Client,
+	volumeSnapshotName string,
+	vsc *snapshotv1api.VolumeSnapshotContent,
+	logger logrus.FieldLogger,
+) error {
+	vs := new(snapshotv1api.VolumeSnapshot)
```

Comment:
I'm curious why VS is retrieved here, but it is not used in the function.
Is it possible to delete the VS from the function?